src/components/navbar/navLinkSection/NavLinkSection.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import classes from './NavLinkSection.module.scss';
+import PropTypes from 'prop-types';
+
+import { openNewsletterSignupModal } from '../.././../utils/functions';
+
+const NavLinkSection = ({ navLinks, toggleDisplayNav }) => {
```

Comment:
Hello! @stellenberger and I were just having a poke around and are a touch confused by this. What is this toggleDisplayNav actually for? We can see it being used (parsed in) anywhere but might just be missing it?

Reply:
So the reason you were getting that error before was it was trying to call the function `null`

Reply:
if you can set it to true over 820px it will probably render with display: none. I have it as a boolean state toggle for the navbar dropdown. I think Arron said something about renaming this before, but I was confused about the comment.